config/schedule.rb

```diff
@@ -0,0 +1,24 @@
+# Use this file to easily define all of your cron jobs.
+#
+# It's helpful, but not entirely necessary to understand cron before proceeding.
+# http://en.wikipedia.org/wiki/Cron
+
+# Example:
+#
+# set :output, "/path/to/my/cron_log.log"
```

Comment:
We should be logging the output somewhere - work with ops to figure out a good place, and if we want it to be global or per-job